rego-templates/embedded.go

```diff
@@ -0,0 +1,101 @@
+package rego_templates
+
+import (
+	"bufio"
+	"embed"
+	"io/fs"
+	"path/filepath"
+	"strings"
+	"sync"
+
+	"github.com/aquasecurity/postee/v2/data"
+	"github.com/aquasecurity/postee/v2/log"
+)
+
+const (
+	COMMON_DIR               = "common"
+	LOCAL_DIR                = "."
+	REGO_PACKAGE_DECLERATION = "package"
+)
+
+var (
+	//go:embed *.rego
+	EmbeddedFiles embed.FS
+	templates     = make(map[string]string)
+	templatesMu   sync.Mutex
+
+	//go:embed common
+	EmbeddedCommonFiles embed.FS
```

Comment:
Could you add some unit test that covers usage of `EmbeddedCommonFiles` and `EmbeddedFiles`